assets/css/hca.scss

```diff
@@ -150,6 +150,25 @@ select {
   }
 }
 
+.hca-review-panel {
+  .usa-button-unstyled {
+    .edit-btn {
```

Comment:
Is there a reason you're using the `.usa-button-unstyled` and overriding the default USWDS button styling?